app/pods/publications/controller.js

```diff
@@ -41,6 +79,28 @@ export default class PublicationsController extends Controller {
     return !this.routing.currentRouteName.startsWith('publications.publication');
   }
 
+  @action
+  async startPublicationFromCaseId(_caseId) {
+    this.set('showLoader', true);
+    // Test if dossier already had publication (index not up to date).
+    const pubFlows = await this.store.query('publication-flow', {
+      filter: {
+        case: {
+          id: _caseId,
+        },
+      },
+      sort: '-created',
+    });
+    let newPublication;
+    if (pubFlows.content.length > 0) {
```

Comment:
Je zou normaal gezien gewoon de length property van de promisearray kunnen aanspreken wanneer hij succesvol geresolved is https://api.emberjs.com/ember-data/3.4/classes/DS.PromiseArray